app/controllers/users_controller.rb

```diff
@@ -297,4 +321,11 @@ def set_tabs(current_tab = "profile")
   def config_changed?
     params[:user].include?(:config_theme)
   end
+
+  def less_than_one_day_old?(user)
+    range = 1.day.ago.beginning_of_day..Time.current
+    user_identity_age = user.github_created_at || user.twitter_created_at || 8.days.ago
```

Comment:
This confused me a bit. Is there a reason we are not also using `user.created` at which should never be `nil` instead of `8.days.ago`?

Reply:
I sort of get that, but to conclusively answer the question is this user's account with us less than a day old, I think `User#created_at` would suffice and be less confusing. But there may be reasons I'm unaware of for doing this via the identity instead.